src/System.Buffers.ReaderWriter/System/Buffers/Reader/BufferReader_text.cs

```diff
@@ -56,9 +159,9 @@ private static ParseDelegate<long> LongParser
         /// <see cref="Utf8Parser.TryParse(ReadOnlySpan{byte}, out bool, out int, char)"/> for supported formats.
         /// </remarks>
         /// <exception cref="FormatException">Thrown if the given format isn't supported.</exception>
-        /// <returns>True if successfully parsed.</returns>
+        /// <returns>Number of consumed bytes if successful, otherwise 0.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static unsafe bool TryParse(ref this BufferReader<byte> reader, out bool value, char standardFormat = '\0')
+        public static unsafe int TryParse(ref this BufferReader<byte> reader, out bool value, char standardFormat = '\0')
```

Comment:
This method doesn't need to be unsafe.

Reply:
Also, do we have precedence elsewhere that a TryX method doesn't return bool? If not, maybe we should rename it.